zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
Yes that seems right too me. I coded something up to that effect this
morning.

I couldn't quite get the test to run without throwing an error.

I will have to take a look tomorrow and see if I can track down the issue.

By the way, is there a guide on setting up an environment for developing
zipline?